demo/apisix/conf/apisix.yaml

```diff
@@ -6,6 +6,31 @@ routes:
       nodes:
       type: roundrobin
     plugin_config_id: 1
+  -
+    id: jmap
+    uri: /jmap
+    service_id: jmap_service
```

Comment:
I tried to add [consumer-restriction](https://apisix.apache.org/docs/apisix/plugins/consumer-restriction/)
plugin, that makes only accepts the `POST` method for `/jmap` uri 
But it didn't work 

Both configs:

```yaml
consumer-restriction:
  allowed_by_methods:
  - methods:
    - "POST"
``` 
and 
```yaml
consumer-restriction:
  allowed_by_methods:
    - "POST"
    
```

Reply:
When creating the route on the dashboard, I see an option to select the methods allowed.

Maybe it can not be configured in standalone mode?

Reply:
Or maybe the document guide is outdated, 
Maybe the using GUI and then read the export config is a way

Reply:
@quantranhong1999 is right. Doc => https://apisix.apache.org/docs/apisix/tutorials/expose-api/

You can see that there is a `methods` argument in the routes where you can put which methods you want to support. Route api doc => https://apisix.apache.org/docs/apisix/admin-api/#route

You misunderstand the goal and usage of consumer-restriction plugin. It's to restrict access to some users, not to define routes. For example, if I have an endpoint `/endpoint` being served with methods GET, POST, PUT, I could say: I want to restrict user `Tung` to only have access to POST method on `/endpoint` and not the other methods.

We obviously don't need that plugin here :)